components/atg/data/Python.ts

```diff
@@ -0,0 +1,48 @@
+import { Type, Directory } from "../Editor/utils/file-manager";
```

Comment:
shouldn't these data be brought from demo-project repo...?

Reply:
No , I made a file-structure for  every language as .ts file in /atg/data folder. Should we fetch them directly from the github repo?

Reply:
yes